Symfony/CS/Fixer/Symfony/BlanklineAfterOpenTagFixer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Ceeram <ceeram@cakephp.org>
+ */
+class BlanklineAfterOpenTagFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+        foreach ($tokens as $index => $token) {
```

Comment:
I can do that. Can you explain why though please?
Op 28 feb. 2015 12:54 schreef "Christophe Coevoet" <notifications@github.com
>:

> In Symfony/CS/Fixer/Symfony/BlanklineAfterOpenTagFixer.php
> <https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1032#discussion_r25556999>
> :
>
> > +use Symfony\CS\AbstractFixer;
> > +use Symfony\CS\Tokenizer\Token;
> > +use Symfony\CS\Tokenizer\Tokens;
> > +
> > +/**
> > + * @author Ceeram <ceeram@cakephp.org>
> > + */
> > +class BlanklineAfterOpenTagFixer extends AbstractFixer
> > +{
> > +    /**
> > +     * {@inheritdoc}
> > +     */
> > +    public function fix(\SplFileInfo $file, $content)
> > +    {
> > +        $tokens = Tokens::fromCode($content);
> > +        foreach ($tokens as $index => $token) {
>
> given that you insert some token in the collection, you should iterate
> backward here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1032/files#r25556999>.
>